express-delete/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.listen(3000, () => {
+});
```

Comment:
Although this will work here it is best for it to be the last thing in the file. Also you should keep the console log in the callback function so you know when your server has started successfully and what port number it is running on.